packages/fluentui/docs/src/prototypes/table/ResponsiveTable.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import { TableCell, TableProps, Table } from '@fluentui/react-northstar';
+import calculateBreakpoints, { Column } from '../../utils/calculateBreakpoints';
+import * as _ from 'lodash';
+
+interface ResponsiveTableProps extends TableProps {
+  config: Column[];
+  children: React.ReactElement<typeof Table>;
+  breakpoints?: number[];
+  id?: string;
+}
+
+const ResponsiveTableContainer: React.FC<ResponsiveTableProps> = props => {
+  const { config, children: ChildrenTable } = props;
+
+  const responsiveTableID = React.useMemo(() => props.id || _.uniqueId(`${Table.className}__responsive-wrapper`), [
+    props.id,
+  ]);
+
+  const [breakpoints, setBreakpoints] = React.useState<{ [key: string]: Column[] }>();
```

Comment:
```suggestion
  const [breakpoints, setBreakpoints] = React.useState<Record<string, Column[]> | undefined>();
```

Reply:
And how the type here is a different from `Breakpoint`? :thinking:

Reply:
Done